### PR TITLE
fixes #13709 - fullscreen editor doesn't need padding in parent element

### PR DESCRIPTION
**Pull request by unorthodoxgeek**

### CODE REVIEW

**Comment by unorthodoxgeek** [comment]
@tbrisker - plz review and/or/xor merge

**Comment by tbrisker** [comment]
@unorthodoxgeek Why not just append to `body` instead of `#main`? 
Also, in the editor version, the navbar has a `margin-top` that should be removed:
screenshot from 2016-02-16 10-42-18
While you are at it, looks like there is some duplicated code here that might be possible to refactor...

**Comment by unorthodoxgeek** [comment]
@tbrisker because it's #main which has the padding, not body.

if I wanted to refactor I'd have moved all the code to the structure I'm pushing. but it's out of scope of the issue.

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -515,6 +515,7 @@ function exit_fullscreen(){
          .resize();
   $('#fullscreen-placeholder').remove();
   $('.exit-fullscreen').remove();
+  $("#main").removeClass("in-fullscreen");
```
Comment (tbrisker) [inline app/assets/javascripts/application.js]: no need for this any more

Comment (unorthodoxgeek) [inline app/assets/javascripts/application.js]: done

**Comment by tbrisker** [comment]
Merged as 9b67a803b88550c1682bf1d2fad8f0a4704e43ec, thanks @unorthodoxgeek!

**Pull request closed without merging**